packages/immutable-arraybuffer/index.js

```diff
@@ -142,6 +142,13 @@ const {
 
 setPrototypeOf(immutableArrayBufferPrototype, arrayBufferPrototype);
 
+/**
+ * Transfer the contents to a new Immutable ArrayBuffer
+ *
+ * @param {ArrayBuffer} buffer The original buffer.
+ * @param {number} [newLength] The start index.
+ * @returns {ArrayBuffer}
+ */
```

Comment:
I retrofitted your improved typing into the main shim, so the two parallel each other using the best of both. Thanks for these improved types.